cognite/neat/rules/models/_rules/information_rules.py

```diff
@@ -292,6 +294,8 @@ def as_dms_architect_rules(self) -> "DMSRules":
             external_id=info_metadata.name.replace(" ", "_").lower(),
             creator=info_metadata.creator,
             name=info_metadata.name,
+            created=datetime.now(),
+            updated=datetime.now(),
```

Comment:
Suggest passing these as optional arguments to this function, and set them to now if they are not set. 

The reason is to make it clear that these are set in the function.